box-ui/src/components/ts/Student/StudentMeeting.tsx

```diff
@@ -290,6 +314,43 @@ const StudentMeeting: FunctionComponent = () => {
             </div>
 
             <div className='sectionButtonsStudent'>
+                <div className='selectProcess'>
```

Comment:
Since this kind of menu is being used in at least two places in the repo it would a good idea to extract it in it's own component.